versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/UseLatestVersionsMojoBase.java

```diff
@@ -74,6 +89,64 @@ public UseLatestVersionsMojoBase(
         super(artifactHandlerManager, repositorySystem, wagonMap, changeRecorders);
     }
 
+    /**
+     * Represents a version change of an artifact
+     */
+    public abstract static class ArtifactVersionChange {
```

Comment:
Should such class be public ... looks like can be use as private

Reply:
Done